daemon/src/daemon.rs

```diff
@@ -114,9 +114,14 @@ impl DocumentActor {
                 .unwrap_or_else(|_| panic!("Could not read file '{persistence_file:?}'"));
             Document::load(&bytes)
         } else {
-            info!("Initializing a new CRDT document");
-            let init = !persistence_file_exists && is_host;
```

Comment:
What if we are the host and the persistence file exists? I think in this else-branch we still want to recreate it, now, because of `--init`.

Reply:
True!